pkgs/development/python-modules/mplfinance/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, matplotlib, pandas, pytest, tox }:
+
+buildPythonPackage rec {
+  pname = "mplfinance";
+  version = "0.12.7a7";
+
+  propagatedBuildInputs = [ matplotlib pandas ];
+  checkInputs = [ pytest tox ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-pATiprTELt8GrEkeKjILPfpdNDVoex5t+Mc+6Gg7cPY=";
+  };
+
+  meta = with lib; {
+    description =
+      "Matplotlib utilities for the visualization, and visual analysis, of financial data";
```

Comment:
Personally not a fan of this line break.

Reply:
`nixfmt` does it to reduce the line length.

Reply:
thats the reason why I switched to nixpkgs-fmt which does not do such line breaks.